tests/migration-util/src/main/java/org/keycloak/test/migration/UpdateAssertsRewrite.java

```diff
@@ -6,29 +6,50 @@
 
 public class UpdateAssertsRewrite extends TestRewrite {
 
+    private final static String ASSERT_TRUE = "assertTrue";
+    private final static String ASSERT_FALSE = "assertFalse";
+    private final static String ASSERT_NULL = "assertNull";
+    private final static String ASSERT_NOT_NULL = "assertNotNull";
+    private final static String ASSERT_EQUALS = "assertEquals";
+    private final static String ASSERT_NOT_EQUALS = "assertNotEquals";
+
     @Override
     public void rewrite() {
         for (int i = 0; i < content.size(); i++) {
             String l = content.get(i);
             String trimmed = l.trim();
-            if (trimmed.startsWith("Assert.")) {
+            if (trimmed.startsWith("Assert.") && trimmed.endsWith(";")) {
                 String method = trimmed.substring("Assert.".length(), trimmed.indexOf("("));
                 int arguments = l.substring(l.indexOf("(") + 1, l.lastIndexOf(")")).split(", ").length;
 
                 if (method.equals("fail")) {
                     directReplace(i, "fail");
-                } else if (method.equals("assertTrue") && arguments == 1) {
-                    directReplace(i, "assertTrue");
-                } else if (method.equals("assertNull") && arguments == 1) {
-                    directReplace(i, "assertNull");
-                } else if (method.equals("assertNotNull") && arguments == 1) {
-                    directReplace(i, "assertNotNull");
-                } else if (method.equals("assertNotNull") && arguments == 2) {
-                    moveMessageToLast(i, "assertNotNull");
-                }  else if (method.equals("assertEquals") && arguments == 2) {
-                    directReplace(i, "assertEquals");
-                } else if (method.equals("assertEquals") && arguments == 3) {
-                    moveMessageToLast(i, "assertEquals");
+                } else if (method.equals("assertThrows") && arguments == 3) {
```

Comment:
@vaceksimon Do you want to make a global var also from this assertThrows?

Reply:
Added a constant for assertThrows and fail